DTS-STN/Service-Canada-Labs
# Changed all instances of Image to img

# Description

[73739 - Update images on site to use html tag](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/?workitem=73739)

The purpose of this PR was to do away with any instances of Next.js' `Image` component in our app and replace them with the html equivalent `<img>`. The reason for this is that while `Next/Image` is supposed to help with image optimization and load times, it has been doing the exact opposite when deployed, either taking a long time to load the images or not at all.

## Acceptance Criteria

`Next/Image` is no longer used in our application in favor of `<img>` tags

## Test Instructions

1. Pull in branch
2. Type `yarn build` & `yarn start`
3. Navigate through the site to ensure images are loading properly


## Help Requested

Our custom `500.js` page currently keeps throwing a client side error when navigating to it. Not really sure why but has nothing to do with this PR as it's on dev as well. Noticed when switching image tags. Might be Next.js related since `404.js` works fine and there aren't really any noticeable differences between how the two are constructed.

## Checklist


## Product and Sprint Backlog

[ADO](https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities/)
